src/events/interactionCreate.ts

```diff
@@ -87,6 +89,26 @@ export const execute = async (interaction: Interaction) => {
           .setDescription(`The tag \`${name}\` has been added successfully`);
         await interaction.reply({ embeds: [embed], ephemeral: true });
       }
+      if (interaction.customId === 'tagEditForm') {
+        if (memberRoles.some((role) => [teamRole, devRole].includes(role))) return
+        const name = interaction.fields.getTextInputValue('tagFormUpdatedName').toLowerCase();
```

Comment:
Is it possible to find a different way to pass in the name? I dont want people to be able to change the name of the tag. This "might" also cause an issue where if the name isnt already in the database it willl throw a random error and **"maybe"** crash the bot.

In theory have them select the name in the command options first then have it check if it exists before letting them edit the infomation

Reply:
Yeah I was going to do it like that and then I didn't and idk why  but yeah I can do it like that